home/vblank.asm

```diff
@@ -34,17 +34,20 @@ VBlank::
 	pop af
 	reti
 
-.VBlanks:
-	dw VBlank0
-	dw VBlank1
-	dw VBlank2
-	dw VBlank3
-	dw VBlank4
-	dw VBlank5
-	dw VBlank6
-	dw VBlank0 ; just in case
-
-VBlank0::
+VBlankHandlers:
```

Comment:
Question; if the pointer table is not really being referenced elsewhere and is located directly after the routine that uses it, should it remain as a local label? Not sure if we have enforced any precedence on this. Just curious on your thoughts.

Reply:
I think it's fine. There aren't many single-colon labels in home/, but there are some, so we don't have an absolute convention of "export every home label" (and probably can/should make more of them local, to be explicit about what *does* get referenced by ROMX).